Cargo.toml

```diff
@@ -72,9 +72,9 @@ optional = true
 default-features = false
 
 [dependencies.syntect]
-version = "4.6.0"
+version = "4.7.0"
 default-features = false
-features = ["parsing"]
+features = ["parsing", "plist-load", "dump-load"]
```

Comment:
If the `dump-load` (or `dump-load-rs`) feature is not specified, the following error is happen. It seems another bug of syntect.

```text
error[E0425]: cannot find function `from_reader` in module `crate::dumps`
   --> /Users/taiki/.cargo/registry/src/github.com-1ecc6299db9ec823/syntect-4.7.0/src/parsing/syntax_set.rs:432:23
    |
432 |         crate::dumps::from_reader(data).expect("data is not corrupt or out of sync with the code")
    |                       ^^^^^^^^^^^ not found in `crate::dumps`
    |
help: consider importing one of these items
    |
1   | use plist::from_reader;
    |
1   | use serde_json::from_reader;
    |

For more information about this error, try `rustc --explain E0425`.
```

Reply:
Ah, this seems to be a breaking change introduced in https://github.com/trishume/syntect/pull/393.

https://github.com/trishume/syntect/blob/b4e2cbb280c79cc5a3b3c12f342d6b792f905001/Cargo.toml#L62-L64

> ```toml
> # If you enable "parsing", you must also enable either "dump-load" or
> # "dump-load-rs", and "dump-create" or "dump-create-rs". Otherwise the code that
> # enables lazy-loading of syntaxes will not compile.
> ```